src/main/kotlin/lotto/domain/WinningCategory.kt

```diff
@@ -2,15 +2,26 @@ package lotto.domain
 
 enum class WinningCategory(val matchCount: Int, val prize: Int) {
     NONE(0, 0),
-    THREE(3, 5000),
-    FOUR(4, 50000),
-    FIVE(5, 1500000),
-    SIX(6, 2000000000),
+    FIFTH(3, 5000),
+    FOURTH(4, 50000),
+    THIRD(5, 1500000),
+    SECOND(5, 30000000),
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    SECOND(5, 30_000_000),
    FIRST(6, 2_000_000_000),
```
가독성을 위해 큰 숫자에는 , 대신 _ 를 이용해 단위를 표기해주기도 합니다 :)